### PR TITLE
cards from Gh8stree

### PR DESCRIPTION
**Pull request by AndreOliveiraMendes**

### CODE REVIEW

File: script/210310305.lua [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    if not e:GetHandler():IsRelateToEffect(e) then return end
+    Duel.Hint(HINT_SELECTMSG,tp,HINTMSG_SPSUMMON)
+    local g=Duel.SelectMatchingCard(tp,c210310305.spfilter2,tp,LOCATION_DECK,0,1,1,nil,e,tp)
+    if g:GetCount()>0 then
+        Duel.SpecialSummon(g,0,tp,tp,false,false,POS_FACEUP)
+    end
+end
+--counter effect
+function c210310305.target(e,tp,eg,ep,ev,re,r,rp,chk)
+   if chk==0 then return Duel.
```
Comment (Radioboy126) [inline script/210310305.lua]: change this time to this my bad :^)  `if` chk==0 then return Duel.IsExistingMatchingCard(Card.IsFaceup, tp, LOCATION_MZONE,LOCATION_MZONE,1,1, nil) end`

File: script/210310305.lua [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+   e1:SetType(EFFECT_TYPE_ACTIVATE)
+   e1:SetCode(EVENT_LEAVE_FIELD)
+   c:RegisterEffect(e1)
+   --special summon
+   local e2=Effect.CreateEffect(c)
+   e2:SetDescription(aux.String210310305(210310305,0))
+   e2:SetCategory(CATEGORY_SPECIAL_SUMMON)
+   e2:SetType(EFFECT_TYPE_FIELD+EFFECT_TYPE_TRIGGER_O)
+   e2:SetProperty(EFFECT_FLAG_DELAY)
+   e2:SetRange(LOCATION_FZONE)
```
Comment (Radioboy126) [inline script/210310305.lua]: LOCATION_SZONE

**Comment by AndreOliveiraMendes** [comment]
done

**Comment by Gideon9212** [comment]
Was approved before hand. Gh8stee will get cont role. Merged.

**Pull request merged**